fedora-kde-min-packages.sh

```diff
@@ -1,6 +1,79 @@
 #!/bin/env bash
-##### CHECK FOR SUDO or ROOT ################################## 
-if ! [ $(id -u) = 0 ]; then echo "This script must be run as sudo or root, try again..."; exit 1 ; fi
+##### CHECK FOR SUDO or ROOT ##################################
+if ! [ $(id -u) = 0 ]; then
+  echo "This script must be run as sudo or root, try again..."
+  exit 1
+fi
 
 # Install KDE Packages
-dnf install NetworkManager-config-connectivity-fedora adwaita-gtk2-theme bluedevil breeze-icon-theme cagibi colord-kde cups-pk-helper dolphin firewall-config glibc-all-langpacks gnome-keyring-pam kcm_systemd kde-gtk-config kde-partitionmanager kde-print-manager kde-settings-pulseaudio kde-style-breeze kdegraphics-thumbnailers kdeplasma-addons kdialog kdnssd kf5-akonadi-server kf5-akonadi-server-mysql kf5-baloo-file kf5-kipi-plugins khotkeys kinfocenter kmenuedit konsole5 kscreen kscreenlocker ksshaskpass ksysguard kwalletmanager5 kwebkitpart kwin pam-kwallet phonon-qt5-backend-gstreamer pinentry-qt plasma-breeze plasma-desktop plasma-desktop-doc plasma-drkonqi plasma-nm plasma-nm-l2tp plasma-nm-openconnect plasma-nm-openswan plasma-nm-openvpn plasma-nm-pptp plasma-nm-vpnc plasma-pa plasma-user-manager plasma-workspace plasma-workspace-geolocation polkit-kde qt5-qtbase-gui qt5-qtdeclarative sddm sddm-breeze sddm-kcm sni-qt xorg-x11-drv-libinput setroubleshoot system-config-users system-config-keyboard system-config-language @"Hardware Support" @base-x @Fonts @"Common NetworkManager Submodules"
+dnf install \
+  @"base-x" \
```

Comment:
I quoted these groups so that they'd be sorted correctly, otherwise `base-x` and `Fonts` would come after `"Hardware Support"`